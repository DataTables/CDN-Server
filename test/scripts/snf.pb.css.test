snf/pb-2.1.0/test.css
200
