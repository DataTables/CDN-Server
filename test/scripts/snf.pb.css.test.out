extensionsURL >snf/pb-2.1.0<
extensionsList > ProductB 2.1.0<
source >{source}<
testString >{testString}<
styling >{styling}<
./ProductB-2.1.0/productb.css


